README.md

```diff
@@ -0,0 +1,34 @@
+[ ![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com)
+
+# Shift React Components
+## Purpose
+
+This repo is all extracted components/objects from the shift-front-end.
```

Comment:
```suggestion
This repository contains a library of React components that are built to be integrated with the data that the SHIFT Platform produces. They should be ready to go out of the box and integrated into any React site that is utilising `shift-node-api`.
```

Just a small excerpt i could think of, but i think it better describes the purpose - we should be mentioning `shift-node-api` to some degree as our props I'd assume are somewhat tied into the objects returned from?